src/meta_merge/core.cljc

```diff
@@ -1,11 +1,16 @@
 (ns meta-merge.core
   (:require [clojure.set :as set]))
 
+(defn metadatable?
```

Comment:
`metadatable?` needs to be replaced, since ClojureScript has `IMeta` and `IWithMeta` protocols. See my earlier comments.